default.nix

```diff
@@ -17,7 +17,7 @@ python3.pkgs.buildPythonApplication rec {
   ];
 
   # does not work in sandbox
-  dontCheck = false;
+  doCheck = false;
```

Comment:
Looks like this flag is also causing checkInputs like pytest to be no longer provided, which breaks the test right now.

Reply:
Also we still define a checkPhase in default.nix. I think this one should be executed still.

Reply:
Then we should remove it because dontCheck does not exist.

Reply:
ok.

Reply:
force pushed